src/components/DeviceItem/DeviceItem.tsx

```diff
@@ -1,61 +1,58 @@
 import { useMemo, useCallback } from "react";
-import { get, find } from "lodash";
-import { Title, TwoProperties } from "@deskpro/app-sdk";
+import { Title, Property, TwoProperties } from "@deskpro/app-sdk";
 import { useExternalLinks } from "../../hooks";
 import { getOS, getHumanCapacity } from "../../utils";
 import { ButtonAsLink, LansweeperLogo } from "../common";
 import type { FC } from "react";
-import type { Maybe } from "../../types";
-import type { Site, Device } from "../../services/lansweeper/types";
+import type { Maybe, DeviceType } from "../../types";
 
 export type Props = {
-  device: Device;
-  sites: Site[];
-  siteId: Maybe<Site["id"]>;
+  device: Maybe<DeviceType>;
```

Comment:
There is no need for this to be `Maybe`. Following the code that calls this, `device` is always set to `DeviceType` so we don't need to complicate this function by catering for things that don't happen.

source: https://github.com/DeskproApps/lansweeper/pull/10/files#diff-ec3c1fdc70706cb1636369651ddafe7432d9d1b2ca05263e75ca0fc1d04fc828R20
source: https://github.com/DeskproApps/lansweeper/pull/10/files#diff-147dc47545e2ab3bc81f3899c5a12fd2b02a87d9f655413e88e542229900310aR51

Reply:
done